searchlib/src/tests/tensor/hnsw_index/hnsw_index_test.cpp

```diff
@@ -305,6 +322,19 @@ TEST_F(HnswIndexTest, 2d_vectors_inserted_in_hierarchic_graph_with_heuristic_sel
     expect_level_0(5, {2, 6});
     expect_levels(6, {{2, 5, 7}, {3}, {}});
     expect_level_0(7, {3, 6});
+    {
+        Slime actualSlime;
+        SlimeInserter inserter(actualSlime);
+        index->get_state(inserter);
+        const auto &root = actualSlime.get();
+        EXPECT_EQ(0, root["memory_usage"]["onHold"].asLong());
+        EXPECT_EQ(8, root["nodes"].asLong());
+        EXPECT_EQ(6, root["valid_nodes"].asLong());
+        EXPECT_EQ(0, root["level_histogram"][0].asLong());
+        EXPECT_EQ(4, root["level_histogram"][1].asLong());
+        EXPECT_EQ(0, root["level_0_links_histogram"][0].asLong());
```

Comment:
Consider checking number of nodes with 1 outgoing link at level 0 as well (root["level_0_links_histogram"][1])